src/lazyload-image.directive.ts

```diff
@@ -24,7 +24,7 @@ export class LazyLoadImageDirective implements OnChanges, AfterContentInit, OnDe
   private scrollSubscription?: Subscription;
   private hooks: HookSet<any>;
 
-  constructor(el: ElementRef, ngZone: NgZone, @Optional() @Inject('options') options?: ModuleOptions) {
+  constructor(el: ElementRef, ngZone: NgZone, @Inject(PLATFORM_ID) private platformId: Object, @Optional() @Inject('options') options?: ModuleOptions) {
```

Comment:
Nice but be consist, it would be nice to skip the `private` keyword here and do something like:

```diff
   private hooks: HookSet<any>;
+  private platformId: Object;

-  constructor(el: ElementRef, ngZone: NgZone, @Inject(PLATFORM_ID) private platformId: Object, @Optional() @Inject('options') options?: ModuleOptions) {
+  constructor(el: ElementRef, ngZone: NgZone, @Inject(PLATFORM_ID) platformId: Object, @Optional() @Inject('options') options?: ModuleOptions) {
     this.elementRef = el;
     this.ngZone = ngZone;
     this.propertyChanges$ = new ReplaySubject();
+    this.platformId = platformId;
     this.hooks = cretateHooks(options);
   }
```